charts/server/templates/sshd/configmap.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
   name: {{ include "zenith.componentname" (list . "sshd-conf") }}
   labels: {{ include "zenith.componentLabels" (list . "sshd-conf") | nindent 4 }}
 data:
-  sshd.yaml: |
+  user-config.yaml: |
```

Comment:
this seems like a really impacting change? Does it have to be?

Reply:
It isn’t. The key just gets changed when you upgrade the chart and the pod gets rotated.